forage-android/src/main/java/com/joinforage/forage/android/LDManager.kt

```diff
@@ -85,4 +95,26 @@ internal object LDManager {
         internalLogger.i("[LaunchDarkly] Vault type set to $vaultType")
         return vaultType as VaultType
     }
+
+    internal fun getPollingIntervalsFeatureFlag(): IntArray {
+        var pollingList = IntArray(0)
+
+        val defaultVal = LDValue.buildObject().put(
+            "intervals",
+            LDValue.Convert.Integer.arrayFrom(listOf(1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000))
+        ).build()
+
+        val jsonIntervals = client?.jsonValueVariation(LDFlags.ISO_POLLING_WAIT_INTERVALS, defaultVal) ?: defaultVal
+
+        val intervals = jsonIntervals.get("intervals")
+
+        // Converting the LDArray into an IntArray
+        // If we cannot access LD, return int array [1000] as a default
+        for (i in 0 until intervals.size()) {
+            pollingList += intervals?.get(i)?.intValue() ?: 1000
```

Comment:
Should be able to remove the `?: 1000` portion now right? `jsonIntervals` always has a non-null value.